maintenance/altering.rst

```diff
@@ -0,0 +1,25 @@
+.. _upgrade:
```

Comment:
Typo in the file name: altering.rst -> alerting.rst

Reply:
At least the typo was consistent :D